src/Iot.Device.Bindings/CompatibilitySuppressions.xml

```diff
@@ -1,6 +1,20 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <!-- https://learn.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids -->
 <Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:BuildHat.Models.BuildHatInformation</Target>
```

Comment:
It's really hard to go through all of these, but I'm assuming they are pretty much all around removing APIs that exposed ImageSharp types, right?

Reply:
There are mainly three kinds of changes: 
- Apis that were changed from ImageSharp to System.Drawing (e.g. `ImageSharp.Color` to `System.Drawing.Color`)
- Apis that were using ImageSharp bitmap types now take `Iot.Device.Common.BitmapImage`
- Individual types that were in incorrect namespaces

For the user, the changes typically only require adapting their using definitions.